lib/std/hash_map.zig

```diff
@@ -1194,7 +1194,7 @@ pub fn HashMapUnmanaged(
         /// fuse the basic blocks after the branch to the basic blocks
         /// from this function.  To encourage that, this function is
         /// marked as inline.
-        inline fn getIndex(self: Self, key: anytype, ctx: anytype) ?usize {
+        fn getIndex(self: Self, key: anytype, ctx: anytype) ?usize {
```

Comment:
Removing `inline` doesn't make sense without the  `@branchHint`. If you'd like, I can make the changes I suggested a separate PR, no problem there.

Reply:
That would be wonderful - I don't have a clear idea of what changing the stdlib like this would do, so you would know better than I. While you thinking of it, would changing similar length assertion branches in other standard types to include @branchHint be a positive change?

Reply:
Possibly. I’ll do it sometime this week, if you could just put back the `inline` that’d be great.

While I’m 97% sure it won’t affect anything, I’ll do a bit more homework by benchmarking before and after.

Reply:
sounds good.